### ISSUE TITLE
Google Search blocks searches for "Auryn alias"

### ISSUE THREAD

**Issue by funkytaco** [rdlowrey/auryn#115]
This is just an FYI. Google Search blocked my searches for "Auryn alias". 

In all of my years of searching I've never seen that. The host they sent me to was https://ipv6.google.com/sorry/IndexRedirect?continue=

If you click that, you will probably see the same thing I did.

**Comment by J7mbo** [comment]
No it doesn't :-)

**Comment by funkytaco** [comment]
Whether it does or not, isn't in debate, as it happened to me.

Note I was on IPV6.

**Comment by rdlowrey** [comment]
Thanks for the report @funkytaco -- I'll look into it and see if I can reproduce with IPv6 ... maybe it was just a momentary blip. Not sure. In any case, I'll check and see if I can't find out anything.

**Comment by funkytaco** [comment]
Just taking a stab in the dark here:

I used to work as a DC Technician, and that's the last time I saw anything like this. Script kiddies had servers that would search Google (from infected servers) for known exploits to spread like a worm usually, but I can't think why this would happen for Auryn. 

Maybe it's related to Win32/Auryn, which is a windows virus.

In other words, it's probably nothing.
